tools/rna_tools/vienna_rna/rnafold.xml

```diff
@@ -292,19 +299,28 @@
             <param name="fasta_input" value="rnafold_input1.fa"/>
             <output name="out_file1" file="rnafold_result1.txt"/>
         </test>
-        
+
         <test>
             <param name="select_fasta" value="true" />
             <param name="fasta_input" value="rnafold_input2.fa"/>
             <param name="temperature" value="75"/>
             <output name="out_file1" file="rnafold_result2.txt"/>
         </test>
-        
+
         <test>
             <param name="select_fasta" value="false" />
             <param name="input_sequence" value="TGGGAATTAGCTCAAATGGTAGAGCGCTCGCTTAGCATGTGAGAGGTAGTGGGATCGATGCCCACATTCTCCA"/>
             <output name="out_file1" file="rnafold_result3.txt"/>
         </test>
+        <test>
+            <param name="select_fasta" value="true" />
+            <param name="fasta_input" value="sample_3.fa"/>
+            <conditional name="shapeOption">
```

Comment:
@bgruening If this test is looking fine, then the entire update looks good to me.